source/_twig-components/functions/faker.function.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+$function = new Twig_SimpleFunction('faker', function ($string, $options="") {
+  $faker = Faker\Factory::create();
+  if ($options != "") {
```

Comment:
We have a default value setting `$options` to empty string. Can't all this just be simplified down to:

`return $faker->$string($options)`

No switch, no check?

Reply:
Probably, I wasn't sure how much effort I should put into this since I was copy/pasting this from an issue queue. Plus the issues with faker generally. Thought of the PR more as a, "this is a route we could go if we can get the Faker plugin fixed."